cypress/fixtures/user.json

```diff
@@ -0,0 +1,27 @@
+{
+  "data": {
+      "id": "1",
+      "type": "user",
+      "attributes": {
+          "email": "ex@ample.com",
+          "name": "Elder Bobby",
+          "date_of_birth": "1999-01-02",
+          "profile_picture": {
+              "name": "profile_picture",
```

Comment:
Was this JSON response made recently? the last merge from the backend should remove all user data from the profile_picture

Reply:
Thanks for letting me know! Just changed it!